src/components/editTestCase/qdm/RightPanel/RightPanel.tsx

```diff
@@ -53,6 +55,23 @@ const RightPanel = ({
         {activeTab === "details" && (
           <DetailsSection canEdit={canEdit} measureName={measureName} />
         )}
+
+        {activeTab === "measurecql" &&
+          (!cqlErrors ? (
```

Comment:
I think might be helpful displaying CQL all the time even there are errors

Reply:
please ignore my above suggestion. QICore is doing the same. I'm not sure why we should hide CQL tab when there are errors.

Reply:
> please ignore my above suggestion. QICore is doing the same. I'm not sure why we should hide CQL tab when there are errors.

Me either. Best I can come up with is maybe we don't have the tooling in place to display those errors in madie-patient.

Reply:
I agree with you Ashok, I don't see an issue with displaying CQL with errors. May be we can take this back to PO's, I do see that we have a error message always on top reminding them that CQL has errors. 

![image](https://github.com/MeasureAuthoringTool/madie-patient/assets/57495404/090f5561-a5ad-4b41-929b-5b1e08a47386)

Reply:
sure, will talk to Christina